dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -26,6 +26,7 @@ RUN apt-get update &&   \
         dmidecode       \
         i2c-tools       \
         psmisc          \
+        python-jsonschema \
```

Comment:
is this python2 version or python3 version? can you change to python3?

Reply:
jsonschema is an implementation of JSON Schema for Python (supporting 2.7+ including Python 3). https://python-jsonschema.readthedocs.io/en/stable/